controllers/velero.go

```diff
@@ -447,6 +461,30 @@ func getFsBackupTimeout(dpa *oadpv1alpha1.DataProtectionApplication) string {
 	return defaultFsBackupTimeout
 }
 
+func getSnapshotMoveDataValue(dpa *oadpv1alpha1.DataProtectionApplication) string {
+	if dpa.Spec.Configuration.Velero != nil && boolptr.IsSetToTrue(dpa.Spec.Configuration.Velero.SnapshotMoveData) {
```

Comment:
In this 2 functions, could we simplify like this?
```go
if dpa.Spec.Configuration.Velero != nil {
    if boolptr.IsSetToTrue(dpa.Spec.Configuration.Velero.<field>) {
        return TrueVal
    }
    // since the field is typed as bool in CRD, there is no other option for the field
    return FalseVal
}
return ""
```

Reply:
The field is a bool pointer so in Go it can have three values - `true`, `false` and `nil` That's why I have done the explicit assignment.

Reply:
you are right, we need to check for `nil`

so, would suggest something like this
```go
if dpa.Spec.Configuration.Velero != nil && value := dpa.Spec.Configuration.Velero.<field>; value != nil {
    if boolptr.IsSetToTrue(value) {
        return TrueVal
    }
    return FalseVal
}
return ""
```

Reply:
`boolptr.IsSetToTrue(value)` already check for nil.